frontend_components/encord_active_components/frontend/src/components/tabs/predictions/PredictionsTab.tsx

```diff
@@ -37,9 +37,16 @@ export function PredictionsTab(
   );
   const [currentTab, setCurrentTab] = useState("0");
 
-  // Auto-select prediction if one exists.
+  // Auto-select prediction if one exists & unselect if an invalid prediction hash is selected;
   useEffect(() => {
-    if (allPredictionOptions != null && allPredictionOptions.length >= 1 && predictionHash == null) {
+    let internalPredictionHash = predictionHash;
+    const a = allPredictionOptions?.findIndex(
+      (v) => v.value === predictionHash
+    );
+    if (a === -1 || a === undefined) {
+      internalPredictionHash = undefined;
+    }
+    if (allPredictionOptions != null && allPredictionOptions.length >= 1 && internalPredictionHash == null) {
```

Comment:
[nitpick] 💅 

I think this is simpler and does the same

```suggestion
    if (
      allPredictionOptions &&
      allPredictionOptions?.find((v) => v.value === predictionHash)
    )
      setPredictionHash(allPredictionOptions[0].value);
```